node/network/availability-distribution/src/lib.rs

```diff
@@ -259,6 +290,18 @@ impl ProtocolState {
 	}
 }
 
+fn cached_live_candidates_unioned<'a>(
```

Comment:
It would be preferable to add a `_inner` suffix.

Reply:
It's not just an "inner" function - I introduced this to get around borrowing issues in another function elsewhere.